src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -222,16 +222,20 @@ trait AttributeComponent {
     }
 
     private def assertConsistentValueListMembers(attributes: Seq[AttributeValue]): Unit = {
-      val headAttribute = attributes.head
-      if (!attributes.forall(_.getClass == headAttribute.getClass)) {
-        throw new RawlsExceptionWithErrorReport(ErrorReport(StatusCodes.BadRequest, s"inconsistent attributes for list: $attributes"))
```

Comment:
these changes allow someone to empty their attribute lists by deleting the members one by one. before you'd get a "next on empty iterator" error when trying to remove the final member of an attribute list